### ISSUE TITLE
Migrating to Angular 2

### ISSUE THREAD

**Issue by mohitt** [joelmukuthu/angular-snapscroll#45]
Hi, 

Is there any plan to migrate to angular 2. 

Regards
Mohit

**Comment by joelmukuthu** [comment]
Hi Monit,

I'm afraid not, I don't have extra time at the moment to look into this. From a quick look at the breaking changes though, with regards to this module, it should be mostly a matter of migrating from the AMD structure. Would you like to have a go at it?

**Comment by athakral** [comment]
Yes, sir, let me try to take care of migrating this to angular 2.0

**Comment by mohitt** [comment]
Yes Sir,  let me try to migrate this to angular 2.0

**Comment by chriszrc** [comment]
@codemerlin is there an angular 2+ version now?
